tests/common/mod.rs

```diff
@@ -5,57 +5,34 @@ use espresso_types::FeeAmount;
 use ethers::prelude::*;
 use futures::future::join_all;
 use reqwest::blocking;
-use std::{
-    fmt,
-    str::FromStr,
-    time::{Duration, Instant},
-};
+use std::{fmt, str::FromStr, time::Duration};
 use surf_disco::Url;
 
 const L1_PROVIDER_RETRY_INTERVAL: Duration = Duration::from_secs(1);
-// TODO add to .env or derive
-const BUILDER_ADDRESS: &str = "0x23618e81E3f5cdF7f54C3d65f7FBc0aBf5B21E8f";
 // TODO add to .env
 const RECIPIENT_ADDRESS: &str = "0x0000000000000000000000000000000000000000";
 /// Duration in seconds to wait before declaring the chain deceased.
 const SEQUENCER_BLOCKS_TIMEOUT: u64 = 120;
 
-async fn wait_for_service(url: &str, interval: u64, timeout: u64) -> Result<String> {
-    future::timeout(Duration::from_secs(timeout), async {
-        loop {
-            if let Ok(body) = blocking::get(format!("{url}/healthcheck")) {
-                return body.text().map_err(|e| {
-                    anyhow!(
-                        "Wait for service, could not decode response: ({}) {}",
-                        url,
-                        e
-                    )
-                });
-            } else {
-                sleep(Duration::from_millis(interval)).await;
-            }
-        }
-    })
-    .await
-    .map_err(|e| anyhow!("Wait for service, timeout: ({}) {}", url, e))?
-}
-
 #[derive(Clone, Debug)]
-struct TestConfig {
-    load_generator_url: String,
-    provider: Provider<Http>,
-    espresso: SequencerClient,
-    builder_address: Address,
-    recipient_address: Address,
-    light_client_address: Address,
-    prover_url: String,
-    builder_url: String,
-    expected_block_height: u64,
-    timeout: f64,
+pub struct TestConfig {
+    pub load_generator_url: String,
+    pub provider: Provider<Http>,
+    pub espresso: SequencerClient,
+    pub builder_address: Address,
+    pub recipient_address: Address,
+    pub light_client_address: Address,
+    pub prover_url: String,
+    pub expected_block_height: u64,
+    pub timeout: f64,
```

Comment:
why is it f64? can we make it Duration?

Reply:
Just to provide that little extra time on the timeout for larger numbers.  `f64` has `.sqrt()` fn. Not sure how we would do the same w/ `Duration`.